cl/api/templates/judge-api-docs-vlatest.html

```diff
@@ -0,0 +1,180 @@
+{% extends "base.html" %}
+{% load extras %}
+
+{% block title %}Judge APIs for Federal and State Judges and Justices – CourtListener.com{% endblock %}
+{% block og_title %}Judge APIs for Federal and State Judges and Justices – CourtListener.com{% endblock %}
+
+{% block description %}Use these APIs to query and analyze thousands of federal and state judges, including their biographical information, political affiliations, education and employment histories, judgeships, and more.{% endblock %}
+{% block og_description %}Use these APIs to query and analyze thousands of federal and state judges, including their biographical information, political affiliations, education and employment histories, judgeships, and more.{% endblock %}
+
+{% block sidebar %}{% endblock %}
+{% block navbar-p %}active{% endblock %}
+
+
+{% block footer-scripts %}
+  {% include "includes/anchors.html" %}
+{% endblock %}
+
+{% block content %}
+<div class="col-xs-12 hidden-md hidden-lg">
+  <h4 class="v-offset-below-2">
+    <i class="fa fa-arrow-circle-o-left gray"></i>
+    <a href="{% url "rest_docs" version="v3" %}">Back to API Docs</a>
+  </h4>
+</div>
+
+
+<div id="toc-container" class="hidden-xs hidden-sm col-md-3">
+  <div id="toc">
+    <h4 class="v-offset-below-3">
+      <i class="fa fa-arrow-circle-o-left gray"></i>
+      <a href="{% url "rest_docs" version="v3" %}">Back to API Docs</a>
+    </h4>
+    <h3>Table of Contents</h3>
+    <ul>
+      <li><a href="#about">Overview</a></li>
+      <li><a href="apis">The APIs</a></li>
+      <ul>
+        <li><a href="#people-endpoint">Judges &amp; Appointers</a></li>
+        <li><a href="#positions-endpoint">Positions Held</a></li>
+        <li><a href="#affiliations-endpoint">Political Affiliations</a></li>
+        <li><a href="#education-history">Educational Histories</a></li>
+        <li><a href="#aba-ratings">ABA Ratings</a></li>
+        <li><a href="#retention-events">Retention Events</a></li>
+        <li><a href="#sources-endpoint">Sources</a></li>
+      </ul>
+    </ul>
+  </div>
+</div>
+
+
+<div class="col-xs-12 col-md-8 col-lg-6">
+  <h1 id="about">Judge and Justice&nbsp;API</h1>
```

Comment:
This narrowing of the database bothers me- but your mileage may vary.  

Could we switch to Judicial Directory API ?

Reply:
Hm, I actually find "directory" to be more narrow, no? It sort of implies that it's just a list of judges, as opposed to a database about them? I'd rather not think too hard about it for now though.

Reply:
sure - im sure itll be fine